addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -340,10 +364,72 @@ const Wysiwyg = Widget.extend({
     redo: function () {
         this.odooEditor.historyRedo();
     },
+    /**
+     * Open the Alt dialog
+     * dialog give access to edit <img> alt and title
+     */
+    openAltDialog(params) {
+        let $baseNode = params.node ? params.node : this.lastTargetedElement;
+
+        let altDialog = new weWidgets.AltDialog(this, params, $baseNode);
+        altDialog.open();
+    },
+    /**
+     * Open the media dialog
+     * use to insert img / icons / videos
+     */
+    openMediaDialog(params) {
+        const range = Wysiwyg.getRange();
+        // we loose the current selection inside the content editable
+        // when we click on the media dialog button
+        // so we need to be able to restore the selection when the modal is closed
+        const restoreSelection = function() {
+            if(range.sc === null) return;
+            Wysiwyg.setRange(range.sc, range.so, range. ec, range.eo);
+        }
+
+        let $baseNode = params.node ? params.node : this.lastTargetedElement;
+        let mediaDialog = new weWidgets.MediaDialog(this, params, $baseNode);
```

Comment:
what about `const`?
Just for a matter of style.

Reply:
use to be variables in the dev process, then forgot to change it back :shrug: